ssg/build_profile.py

```diff
@@ -14,7 +15,8 @@
 from .constants import anaconda_system as anaconda_rem_system
 from .constants import cce_uri
 from .constants import ssg_version_uri
-from .constants import stig_ns, cis_ns, hipaa_ns, anssi_ns, ospp_ns, cui_ns
+from .constants import stig_ns, cis_ns, generic_stig_ns, hipaa_ns, anssi_ns
```

Comment:
Does uh, the STIG NS actually _do_ anything? 

```bash
$ gif 'https://public.cyber.mil/stigs/srg-stig-tools/'
ssg/constants.py:63:stig_ns = "https://public.cyber.mil/stigs/srg-stig-tools/"
```

The only place I see it is in the constants...

In the generated RHEL8 XCCDF I see:

```
          <reference href="https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux">RHEL-08-040124</reference>
```

Which is the new `generic_stig_ns` I introduced.... Seems like RHEL 8 needs to update its constant?